.pre-commit-config.yaml

```diff
@@ -1,22 +0,0 @@
-repos:
-- repo: https://github.com/sirosen/check-jsonschema
-  rev: 0.8.2
-  hooks:
-    - id: check-jsonschema
-      name: "Validate JSON files against JSON Schemas"
-      files: building_inputs/master_general_resources_database.json
-      types: [json]
-      args: ["--schemafile", "json_schemas/master_general_resources_database.json"]
-- repo: https://github.com/psf/black
-  rev: 21.12b0
-  hooks:
-    - id: black-jupyter
-      name: "Lint Jupyter notebooks using Black"
-      files: decision-support-tool.ipynb
-- repo: https://github.com/mwouts/jupytext
-  rev: v1.11.0
-  hooks:
-    - id: jupytext
-      name: "Generate a .py file from the notebook"
-      entry: jupytext --to py decision-support-tool.ipynb
-      pass_filenames: false
```

Comment:
Please keep this file. If you absolutely need to deactivate, simply uninstall `pre-commit` from your local machine. This validation might be sometimes annoying for small changes/wip, but before deploying it's really important to make sure nothing broke and we don't have regression. I would say that you can bypass this system for `feat-branch -> develop`, but for `develop -> master` I prefer to keep this mecanism.

Reply:
Thanks for these.  Will revert these changes, and work on workflow on my end as per your comments.